packages/api/src/api/store.ts

```
@@ -0,0 +1,100 @@
+   * @secure
+   */
+
+  async getInventory<T = Record<string, number>>() {
+    try {
+      const method = "get";
+      const endpoint = "/store/inventory";
+      const options = undefined;
+
+      return await apiInstance[method]<T>(endpoint, options);
+    } catch (error) {
+      throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)